MatrixSDK/Data/MXRoomSummaryUpdater.m

```diff
@@ -516,8 +516,7 @@ - (BOOL)updateSummaryDisplayname:(MXRoomSummary *)summary session:(MXSession *)s
             }
         }
 
-        if (memberCount > 1
-            && (!displayName || [displayName isEqualToString:_roomNameStringLocalizer.emptyRoom]))
+        if (!displayName || [displayName isEqualToString:_roomNameStringLocalizer.emptyRoom])
```

Comment:
I think we should keep here a sanity check on memberCount
`memberCount > 0` 
because of the computation based on this count in `fixUnexpectedEmptyRoomDisplayname`

Reply:
@giomfo in the particular case, `memberCount` is initialized to 0 and updated only if there is a `serverRoomSummary` (which is not the case) or if `roomState.membersCount.members > 1` (which is also not the case).
So the value of `memberCount` will be 0, which is not a problem for the `fixUnexpectedEmptyRoomDisplayname` method because there are no other members in the room, so `memberNames.count` will be 0 and the fix will apply.

Reply:
@nimau I'm ok about the targeted use case. So remove this sanity check here is ok for me
But don't you think we would add some sanity check in `fixUnexpectedEmptyRoomDisplayname` where we reduce by 1 or 2 the provided `memberCount`?